index.html

```diff
@@ -2309,6 +2309,97 @@ <h2 id="AudioDestinationNode">
+              between the moment the implementation sends a buffer to the
+              system and the audible sound from the audio output (speaker or
+              connector).
+            </p>
+          </dd>
+          <dt>
+            AudioTimestamp&lt;void&gt; getPosition()
+          </dt>
+          <dd>
+            <p>
+              Returns <a><code>AudioTimestamp</code></a> instance containing
+              two correlated context's audio stream position values: the
+              <a><code>contextTime</code></a> member contains estimation of the
+              audio stream position which is currently being rendered by the
+              <a><code>AudioDestinationNode</code></a> (i.e. the current output
+              audio stream position), in the same units and origin as context's
+              <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>;
+              the <code><a>performanceTime</a></code> member contains
+              performance time estimation of the moment when the stored audio
+              stream position is rendered, in the same units and origin as
+              <code>performance.now()</code> (described in [[!hr-time-2]]).
+            </p>
+            <p>
+              If the context's rendering graph has not yet processed a block of
+              audio, then <a><code>getPosition</code></a> returns
+              <code>AudioTimestamp</code> instance with both members containing
+              zero.
+            </p>
+            <p>
+              The <a><code>getPosition</code></a>'s result can be used to get
+              performance time estimation for the slightly later context's time
+              value:
+            </p>
+            <pre class="highlight example">
+            function outputPerformanceTime(contextTime) {
+                var timestamp = destNode.getPosition();
+                var elapsedTime = contextTime - timestamp.contextTime;
+                return timestamp.performanceTime + elapsedTime * 1000;
+            }
+            </pre>
+            <p>
+              The closer input value to <code>timestamp.contextTime</code>, the
+              better preciseness of the obtained estimation.
+            </p>
+            <p class="note">
+              The difference between the values of context's <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>
+              and the <a href=
+              "#widl-AudioTimestamp-contextTime"><code>contextTime</code></a>
+              returned from <a href=
+              "#widl-AudioNode-getPosition"><code>getPosition</code></a> call
+              cannot be considered as a reliable output latency estimation
+              because <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>
+              is incremented in jumpy way and at unpredictable moments, so
```

Comment:
"is incremented in jumpy way and at unpredictable moments" -> "may be incremented at non-uniform time intervals"